templates/Dockerfile.template

```diff
@@ -66,6 +82,19 @@ RUN mkdir -p /data/db /data/configdb \
 VOLUME /data/db /data/configdb
 
 COPY docker-entrypoint.sh /entrypoint.sh
+RUN chmod +x /entrypoint.sh
```

Comment:
This is not needed. Please check executable bit on those files and `umask` in your environment.

Reply:
done